static/sass/layout/_nav.scss

```diff
@@ -1,23 +1,100 @@
 // Site Navigation
 // ===============
 
-$nav-locations: 'birds', 'work', 'speaking', 'open-source', 'blog', 'community', 'contact';
+$locations: 'projects', 'open-source', 'blog', 'birds', 'speaking', 'community', 'contact';
+$nav-locations: 'work', 'about';
 
 .nav-link {
+  color: color('action-alt');
+  display: inline-block;
+  font-weight: bold;
+  padding: size('half-shim') size('shim');
+  text-decoration: none;
+  white-space: nowrap;
+
+  @include above('nav-narrow') {
+    padding: size('shim') size('gutter');
+  }
+
   #{$link} {
-    display: inline-block;
-    padding: size('half-shim');
-    text-decoration: none;
+    color: color('action-alt');
+  }
+
+  #{$focus} {
+    color: color('action');
+  }
+
+  &[aria-pressed='true'] {
+    color: color('action');
   }
 }
 
-@each $location in $nav-locations {
+@each $location in $locations {
   [data-location='#{$location}'] {
     [data-active='#{$location}'] {
-      #{$link} {
-        color: color('action');
-        font-style: italic;
-      }
+      color: color('action');
+    }
+  }
+}
+
+@each $location in $nav-locations {
+  [data-nav-location='#{$location}'] {
+    [data-active='#{$location}'] {
+      color: color('action');
+    }
+  }
+}
+
+.nav {
+  position: relative;
+}
+
+.main-nav {
+  display: flex;
+  justify-content: space-between;
+  margin: negative('gutter') auto 0;
+  max-width: size('nav-narrow');
+  padding: 0 size('shim');
+  width: 100%;
+
+  > li {
+    display: inline-flex;
+
+    &:nth-of-type(2) {
+      flex-grow: 1;
+    }
+
+    @include above('nav-medium') {
+      position: relative;
     }
   }
 }
+
+.is-dropdown {
+  background-color: color('background');
+  border-bottom: size('border') solid color('border');
+  box-shadow: 0 6px 5px color('shadow');
+  left: 0;
+  padding: size('gutter') + size('half-shim') 0 size('shim');
+  position: absolute;
+  right: 0;
+  top: 100%;
+  width: 100vw;
+  z-index: 1;
+
+  .nav-link {
```

Comment:
Maybe split `.nav-link` into two classes (`.nav-link-main` and `.nav-link-sub`) instead of overriding it in different contexts?